packages/components/view-switcher/README.md

```diff
@@ -1,26 +1,28 @@
+<!-- THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY. -->
```

Comment:
Thanks. I wondered actually why it wasn't generated when changing it.

Reply:
The package must have the `docs` folder, otherwise the generation is skipped.